documentation/src/main/asciidoc/ch11.asciidoc

```diff
@@ -123,6 +122,64 @@ required, a deterministic evaluation order can be enforced using group sequences
 Refer to <<section-provider-specific-settings>> to learn about the different ways of enabling the
 fail fast mode when bootstrapping a validator.
 
+[[section-method-validation-prerequisite-relaxation]]
+=== Relaxation of requirements for method validation in class hierarchies
+
+The Bean Validation specification defines a set of preconditions which apply when defining
+constraints on methods within class hierarchies. These preconditions are defined in
+http://beanvalidation.org/1.1/spec/#constraintdeclarationvalidationprocess-methodlevelconstraints-inheritance[section 4.5.5]
+of the Bean Validation 1.1 specification. See also <<section-method-constraints-inheritance-hierarchies>>
+in this guide.
+
+As per specification a Bean Validation provider is allowed to relax these preconditions.
+With Hibernate Validator you can do this in one of two ways.
+
+First you can use the configuration properties _hibernate.validator.allow_parameter_constraint_override_,
+_hibernate.validator.allow_multiple_cascaded_validation_on_result_ and
+_hibernate.validator.allow_parallel_method_parameter_constraint_ in _validation.xml_. See example
+
+
+[[example-relaxing-method-validation-xml]]
+.Configuring method validation behaviour in class hierarchies via properties
+====
+[source, XML]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<validation-config
+        xmlns="http://jboss.org/xml/ns/javax/validation/configuration"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://jboss.org/xml/ns/javax/validation/configuration validation-configuration-1.0.xsd">
+    <default-provider>org.hibernate.validator.HibernateValidator</default-provider>
+
+    <property name="hibernate.validator.allow_parameter_constraint_override">true</property>
+    <property name="hibernate.validator.allow_multiple_cascaded_validation_on_result">true</property>
+    <property name="hibernate.validator.allow_parallel_method_parameter_constraint">true</property>
+</validation-config>
+----
+====
+
+Alternatively these settings can be applied during programmatic bootstrapping using a
+`MethodValidationConfiguration` instance.
+
+[[example-relaxing-method-validation]]
+.Configuring method validation behaviour in class hierarchies
+====
+[source, JAVA]
+----
+HibernateValidatorConfiguration configuration = Validation.byProvider(HibernateValidator.class).configure();
+
+MethodValidationConfiguration mvc = new MethodValidationConfigurationImpl();
+mvc.allowMultipleCascadedValidationOnReturnValues(true);
+mvc.allowOverridingMethodAlterParameterConstraint(true);
+mvc.allowParallelMethodsDefineParameterConstraints(true);
+
+configuration.setMethodValidationConfiguration(mvc);
+----
+====
+
+By default, all of these properties are false, implementing the default behavior as defined in the
```

Comment:
I'd like us to add a warning that the defaults are what they are for a good reason, so one should be very careful and have a good reason when changing them, as this may come to the surprise of other developers.